src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.Tasks;
+import tasks.ToDo;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String BAR =
+            "    ____________________________________________________________";
+
+    private static final String INDENTATION = "     ";
+    private static final String NEW_LINE = "\n";
+
+    private static void greet() {
+        System.out.println(BAR);
+        System.out.println(INDENTATION + "Hello! I'm Duke");
+        System.out.println(INDENTATION + "What can I do for you?");
+        System.out.println(BAR);
+        System.out.print(NEW_LINE);
```

Comment:
Would it be more intuitive to use `System.out.println()` with no arguments instead?